.travis.yml

```diff
@@ -74,12 +80,19 @@ addons:
       - g++-4.7
       - gcc-4.9
       - g++-4.9
+      - gcc-6
+      - g++-6
 
 before_install:
   - echo $TRAVIS_NODE_VERSION
   - npm config set python `which python`
   - if [ $TRAVIS_OS_NAME == "linux" ]; then
-      if [[ $(node -v) =~ v[1-9][0-9] ]]; then
+      if [[ $(node -v) =~ v13 ]]; then
```

Comment:
According to BUILDING.md we should be using gcc 6 also for node 12... II think we should change this